src/monkey_patch/language_models/bedrock_api.py

```diff
@@ -0,0 +1,124 @@
+import abc
+import json
+
+import boto3 as boto3
+import openai
+import time
+# import abstract base class
+from monkey_patch.language_models.llm_api_abc import LLM_Api
+import os
+
+OPENAI_URL = "https://api.openai.com/v1/chat/completions"
+import requests
+
+AWS_REGION = os.environ.get("AWS_DEFAULT_REGION", "us-east-1")
```

Comment:
We should instantiate all envvars / bedrock specific logic inside the `init` of the Bedrock class - catching and reraising any thrown exceptions.